src/app/reset-password/reset-password/reset-password.component.html

```diff
@@ -197,14 +197,6 @@
                 </mat-error>
               </mat-form-field>
 
-              <div
-                *ngIf="success === true"
-                class="error-container"
-                i18n="@@account.yourPasswordHasBeenUpdate"
```

Comment:
@leomendoza123 Could you please also remove the unused property from the `*.properties` files